crowdbotics/modules
# PLAT-5612 Add fastlane commands for processing iOS privacy policy

## Ticket

PLAT-5612
_Related tickets:_ PLAT-5590, PLAT-5589
_Related PRs:_ #3139

## Type of PR


## Changes introduced

- An update to the Fastfile configuration where it uploads the `app_privacy_details.json` to app store
- Regenerated the scaffold using `npm run bootstrap` to include the updates

## Test and review

- I have tested this manually through a dummy app store app using my own app store account to check if the privacy policy options are updated as described in the json
- The Appetize deployments succeeded with the changes

1. Create a react native app
2. Deploy an app
3. Change iOS Privacy Options under settings
4. Deploy again
5. The Privacy Options on the App Store App should be updated accordingly

This can probably not be tested on App Store for free apps (cause they're deployed to Appetize) so we'll need to create a dummy app that's deployed to App Store so we can check that it works properly
